src/serializer/ResidualHeapSerializer.js

```diff
@@ -229,6 +231,7 @@ export class ResidualHeapSerializer {
   additionalFunctionValueNestedFunctions: Set<FunctionValue>;
 
   generatorParents: Map<Generator, Generator | FunctionValue | "GLOBAL">;
+  conditionalFeasibility: Map<AbstractValue, { t: boolean, f: boolean }>;
```

Comment:
conditionalFeasibility , t and f are not great names. There should also be a comment here explaining what this map represents.

Reply:
Will add comment.